### DISCUSSION TITLE
To Do: Issue Template

### DISCUSSION THREAD

**Discussion by coreywillwhat** [audiconnect/audi_connect_ha#355] [Ideas]
Users are reporting issues with no logs and very minimal details. Makes it hard to move things along as we have to keep asking for details and walking them through how to get logs, and waiting.

Putting this as a placeholder to create an issue template for users when they submit an issue. If someone else has time to get to it that would be great too.

**Comment by Kolbi** [comment]
#363 

Feature template also? Seems I need to create a second branch for it :/

**Reply by coreywillwhat** [reply]
Thanks for doing these!
